### ISSUE TITLE
One annotation change corrupts entire model

### ISSUE THREAD

**Issue by python3Berg** [explosion/spaCy#7162]
Hello,
I have a custom named entity model with 000's of annotations that has been incrementally trained for months. I am using spacy2.2, python3 and my own training script originally based on your documentation.

If I remove a single annotation, and replace it with one or more different labels that overlap with the same token-start and end, my models, on incremental training paths, will show losses increased by 100-200x... 5x larger that a clean model at start.

Since I have little control over changes that come from client text annotations, this seems like an unexpected outcome. I could see where changes and subsequent relearning would require some time, but a single change seems to corrupt everything.

**Comment by svlandeg** [comment]
Could you share some sample output? Is it just the training loss that jumps so strongly, or also the accuracy on the dev set?

**Comment by python3Berg** [comment]
Truthfully, I need to do some work to answer question. Once my model was mature, I streamlined some of the testing and used loss as a proxy. As my annotation set incrementally grows, the increase in accuracy was so slow as to be uninteresting. Seeing the loss jump so dramatically, and not reduce even after dozens of iterations, made me think it unlikely that other metrics would be unaffected.

**Comment by svlandeg** [comment]
I'll go ahead and move this to the discussion forum in the meantime, as this type of discussion is perfectly suited for that. 

We haven't really encountered the behaviour you described before, so it would be good to get a sample output / some more background / some reproducible code snippet to be able to look into this further!
